### PR TITLE
[MONIT-29238] Removed resteasy-jaxrs dependency

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

File: java-lib/pom.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>java-lib</artifactId>
-  <version>2022-08.3-SNAPSHOT</version>
+  <version>MONIT-29238-resteasy-jaxrs-SNAPSHOT</version>
```
Comment (joannak-vmware) [inline java-lib/pom.xml:6]: Should this be changed back to `2022-08.3-SNAPSHOT`?

File: yammer-metrics/pom.xml [hunk 2]
```diff
@@ -3,12 +3,12 @@
     <parent>
         <artifactId>javalib</artifactId>
         <groupId>com.wavefront</groupId>
-        <version>2022-08.3-SNAPSHOT</version>
+        <version>MONIT-29238-resteasy-jaxrs-SNAPSHOT</version>
```
Comment (joannak-vmware) [inline yammer-metrics/pom.xml:6]: Same as above, should this be `2022-08.3-SNAPSHOT`?

**Review by joannak-vmware — APPROVED** [review]
See comment above, please verify java-lib version SNAPSHOTs. Otherwise LGTM

**Comment by laullon** [comment]
we will change the version before release

**Pull request merged**